boat-scaffold/src/main/templates/boat-android/build.gradle.mustache

```diff
@@ -94,44 +91,18 @@ buildscript {
 allprojects { project ->
     repositories {
         google()
+        mavenCentral()
+        jcenter()
         // These variables must be set up on your machine's global gradle.properties:
-        maven {
-            url = 'https://artifacts.backbase.com/android3'
-            credentials {
-                username = mvnUser
-                password = mvnPass
-            }
-        }
-        maven {
-            url = 'https://artifacts.backbase.com/android-retail3'
-            credentials {
-                username = mvnUser
-                password = mvnPass
-            }
-        }
-        maven {
-            url = 'https://artifacts.backbase.com/android-wealth3'
-            credentials {
-                username = mvnUser
-                password = mvnPass
+        ["android3", "android-retail3"].forEach { repo->
```

Comment:
This should align with

https://backbase.atlassian.net/wiki/spaces/GUIL/pages/4365058337/Mobile-specific+info+Android

Example is in Golden Sample App